app/models.py

```diff
@@ -200,7 +200,8 @@ class Agency:
   # An agency which had at least 1 eligible domain.
   def eligible(report_name):
     return db.table('agencies').search(
-      Query()[report_name]['eligible'] > 0
+      (Query()[report_name]['eligible'] > 0) |
+      (Query()[report_name]['pages_count'] > 0)
```

Comment:
Heh, because I wasn't thinking. The previous process_a11y script didn't use "eligible" and I was keeping things the same to make sure that if folks were using the previous report, things wouldn't break. But there's probably no reason not to just add "eligible" to the report as well, containing the same data.